downloads/az/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,174 @@
+
+Göstərilən budağı silir
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colOne | markdownify }}
+</div>
+
+
+{% capture colTwo %}
+
+## Repozitarın yaradılması
+
+Yeni bir repozitar ilə işə başlamaq üçün, bunu ya lokalda yaradıb sonra GitHub-a göndərməli və ya mövcud repozitarı klonlaşdırmalısınız. Hər repozitar üçün bu əməliyyatı sadəcə bir dəfə edəcəksiz.
+
+```$ git init```
+
+Mövcud bir qovluğu Git repozitarına çevirir
+
+```$ git clone [url]```
+
+GitHub-da mövcud olan repozitarı, bütün faylları, budaqları və kommitləri daxil olmaqla klonlaşdırır (yükləyir).
+
+## .gitignore faylı
+Bəzən, müəyyən faylları Git tərəfindən izlənmədən istisna etmək əlverişli ola bilər. Bu, adətən `.gitignore` adlı xüsusi bir faylda edilir. `.gitignore` faylları üçün faydalı şablonları [github.com/github/gitignore](https://github.com/github/gitignore) ünvanında tapa bilərsiniz.
+
+## Dəyişikliklərin sinxronizasiyası
+
+Lokal repozitarınızı Github.com-da olan repozitarınız ilə sinxronlaşdırın
+
+```$ git fetch```
+
+GitHub-da mövcud olan repozitarın bütün tarixçəsini yükləyir
+
+```$ git merge```
+
+GitHub-da mövcud olan repozitarı cari budaqla birləşdirir
+
+```$ git push```
+
+Lokal budaqda olan bütün kommitləri Github repozitarına göndərir
+
+```$ git pull```
+
+Cari işlək lokal budağı müvafiq Github budağından gələn bütün yeni kommitlərlə yeniləyir. `git pull` - `git fetch` və `git merge` əmrlərinin kombinasiyasıdır
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colTwo | markdownify }}
+</div>
+<div class="clearfix"></div>
+
+{% capture colThree %}
+
+## Dəyişikliklər edin
+Dəyişikliklərə baxış və kommitlərin yaradılması
+
+```$ git status```
+
+Bütün yeni və ya dəyişdirilmiş faylların siyahısını sadalayır
+
+```$ git log```
+
+Cari budaq üçün versiya tarixini siyahılayır
+
+```$ git log --follow [fayl]```
+
+Bir fayl üçün ad dəyişmələri də daxil olmaqla versiya tarixini siyahılayır
+
+```$ git diff [birinci-budaq]...[ikinci-budaq]```
+
+İki budaq arasındakı məzmun fərqlərini göstərir
+
+```$ git show [commit]```
+
+Müəyyən bir kommit üçün metadata və məzmun dəyişikliklərini göstərir
+
+```$ git add [file]```
+
+Göstərilən faylı sonrakı əməliyyatlar üçün indeksləyir
+
+```$ git commit -m "[izahlı mesaj]"```
+
+İndekslənmiş dəyişiklikləri versiya tarixinə yazır
+
+## Kommitlərin geri qaytarılması
+
+Səhvlərin silinməsi və tarixin tənzimlənməsi
+
+```$ git reset [kommit]```
+
+Bütün dəyişiklikləri işçi qovluğunda saxlayaraq, göstərilən `[commit]`-dən sonrakı kommitlərin hamısını ləğv edir. 
+
+```$ git reset --hard [kommit]```
+
+Bütün tarixi, işçi qovluğu daxil olmaqla ləğv edir və göstərilən kommitə qaytarır.
+
+> DİQQƏT! Tarixi dəyişdirmək xoşagəlməz yan təsirlərə səbəb ola bilər. GitHub-da mövcud olan kommitləri dəyişdirərkən ehtiyatlı olun. Əgər köməyə ehtiyacınız varsa, github.community və ya dəstəklə əlaqə saxlayın.
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colThree | markdownify }}
+</div>
+
+{% capture colFour %}
+
+## Lüğət
```

Comment:
If you are adding a glossary, I think it will be useful to also include the translations that you did throughout the page. Example:

- **branch** (budaq): ....

Reply:
Good idea